benchmark/matmul.cpp

```diff
@@ -325,7 +388,7 @@ static void NvFuserScheduler_Matmul_8warp4stage(
       ->Args({784, 72, 8})                                                   \
       ->Args({784, 8, 72}) /*->Args({1, 1, 2048})*/                          \
       ->Args({1024, 1024, 1024})                                             \
-      ->Unit(benchmark::kMicrosecond)                                        \
+      ->SplitKMatmulA100Shapes->Unit(benchmark::kMicrosecond)                \
```

Comment:
As the name suggests, these are the shapes pulled from TIMM. I think this should not include any split-k shapes?

Reply:
Agreed. This was just a hack. I'll split them off.